site/content/3.13/data-science/arangographml/arangograph-ml.md

```diff
@@ -0,0 +1,183 @@
+---
+title: ArangoGraphML Web Interface
+menuTitle: ArangoGraphML Web Interface
+weight: 15
+description: >-
+ Enterprise-ready, graph-powered machine learning as a cloud service or self-managed
+aliases:
+  - getting-started-with-arangographml
+---
+Solve high-computational graph problems with Graph Machine Learning. Apply ML on a selected graph to predict connections, get better product recommendations, classify nodes, and perform node embeddings. Configure and run the whole machine learning flow entirely in the web interface.
+
+## Creating a GraphML Project
+
+To create a new GraphML project using the ArangoDB Web Interface, follow these steps:
+
+- **Select the Target Database** – From the **Database** dropdown in the left-hand sidebar, select the database where the project should reside.
+- **Navigate to the Data Science Section** – In the left-hand navigation menu, click on Data Science to open the GraphML project management interface, then click on RunGraphML.
+![Navigate to Data Science](../../../images/datascience-intro.jpg)  
+- **Click "Add new project"** – In the **GraphML projects** view, click  **Add new project**.
```

Comment:
The style seems to be a list items having a concise description of the step in bold and then the concrete actions following after a hyphen (non-bold). Here, this pattern is broken because a specific UI element is referenced in the first part. Please make this uniform. I'm not sure how much value this adds, perhaps we can replace the first part with an introduction above the text to outline the procedure.